src/core/primitives.js

```diff
@@ -378,6 +378,12 @@ class RefSetCache {
   clear() {
     this._map.clear();
   }
+
+  *refValues() {
```

Comment:
Could you add a unit test to the primitives test suite to exercise this code? It might be similar to https://github.com/mozilla/pdf.js/blob/master/test/unit/primitives_spec.js#L496-L500 which tests the iterator above.